### PR TITLE
Readme&intro

### PR DESCRIPTION
**Pull request by HY-D1**
finish readme file, and start brief introduction, but explains about data set and topic is not clear enough, background almost finish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+Use dataset from open source websites [UCI Machine Learning Repository: Data Sets](https://archive.ics.uci.edu/dataset/332/online+news+popularity) to predict the number of shares of a article. We split the data into traing and testing parts, and proceed full model and reduced model to make the results more reliable and efficent enough.
+
+### Dependencies
+
+To finish up this project you will need to install:
+
+-   R programming language
+-   R packages:
+    -   `renv`
+    -   `GGally`
```
Comment (agill59) [inline README.md:20]: Renv will deal with the packages, so only R and Quarto are needed.

**Review by agill59 — CHANGES_REQUESTED** [review]
If you could remove the list of R packages, as well as put the intro inside the .qmd file instead of the .ipynb

File: README.md (hunk 1 above)
Comment (HY-D1) [reply README.md:20]: ok, i will do this

**Review by agill59 — APPROVED** [review]
Looks good, I will convert your references to BibTex so we can have a bibliography file.

**Pull request merged**